gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/desktop/application/tasks/AbstractJPackageTask.kt

```diff
@@ -287,7 +287,8 @@ abstract class AbstractJPackageTask @Inject constructor(
 
             val mappedJar = libsMapping[launcherMainJar.ioFile]?.singleOrNull()
                 ?: error("Main jar was not processed correctly: ${launcherMainJar.ioFile}")
-            cliArg("--main-jar", mappedJar)
+            val mainJarRelative = (if (currentTarget.os == OS.Windows) "\\" else "/") + mappedJar.relativeTo(libsDir.ioFile).toString()
```

Comment:
I believe that the leading slash (`"\\"` or `"/"`) is unnecessary and in fact, on Windows, that leading slash is causing `jpackage` to fail. See screenshot:

![Capture](https://user-images.githubusercontent.com/2899129/226139303-df04e508-4a26-4690-ade5-202f202904a7.PNG)

Notice in the above screenshot that the first command failed, but the second one succeeded (since the leading slash was omitted).